.gitignore

```diff
@@ -20,3 +20,4 @@ bin/*
 !bin/install
 !bin/sylius-cs-fixer
 vendor
+/.idea/
```

Comment:
We do not add IDE directories to gitignore (it would become too messy if everyone adds his IDE!), could you add it to .git/info/exclude on your local repository and revert this change? Thank you!